backends/s3/client.go

```diff
@@ -75,6 +76,16 @@ func (c *client) Backup() error {
 	return c.putObject(bytes.NewReader(data), c.Bucket, backupKey)
 }
 
+func (c *client) Lock() error {
+	// TODO: Implement me!
```

Comment:
@bndw maybe you find the time to implement this? Please merge this PR asap and fix in a separate PR :)